spec/support/runner.rb

```diff
@@ -368,8 +382,15 @@ def setup_commands
       ]
     end
 
-    def run_command
-      "APPSIGNAL_APP_ENV=test node_modules/.bin/appsignal-diagnose #{@arguments.join(" ")}"
+    def run_env
+      super.merge({
+        "NODE_ENV" => "test",
+        "APPSIGNAL_APP_ENV" => "test"
```

Comment:
```suggestion
        "NODE_ENV" => "development",
        "APPSIGNAL_APP_ENV" => "development"
```

I prefer development here, because we're not running the apps as a test env usually when people are actually running the diagnose tool.

And does AppSignal not inherit the env from the `NODE_ENV` variable? I have no idea actually.

Reply:
:+1: to changing it, but then we should change it everywhere, not just for Node.